docs/Getting Started/Debian/Debian Bullseye Root on ZFS.rst

```diff
@@ -255,7 +255,7 @@ Step 2: Disk Formatting
 
    - If you are creating a mirror topology, create the pool using::
 
-       zpool create \
+       zpool create -f \
```

Comment:
See above: You don't want garbaged filesystem, just make clear, that you don't use this partition